Makefile

```diff
@@ -274,11 +177,14 @@ clean:
 	-rm -rf EXM_*/synthea_output
 	-rm .setup-cqf-ruler-stu3
 	-rm .setup-cqf-ruler-r4
+	-rm .setup-cqf-ruler
 	-rm .new-cqf-ruler
 	-rm .seed-measures-stu3
 	-rm .seed-measures-r4
+	-rm .seed-measures
 	-rm .seed-vs-stu3
 	-rm .seed-vs-r4
+	-rm .seed-vs
 	-rm .copy-valuesets
```

Comment:
I know this isn't your doing, but it makes sense to change it here I think.

We don't have rm -rf connectathon in the cleanup here, so it needs to be manually removed when we update the connectathon hash as you've done here.

Mind adding this to the make clean? I don't see any drawbacks

Reply:
I added a `make deep-clean` task to support this, as well as cleaning the `synthea` repo.